packages/directlinespeech/__tests__/utilities/fetchSpeechCredentialsWithCache.js

```diff
@@ -32,19 +45,44 @@ async function fromSubscriptionKey({ region, subscriptionKey }) {
   };
 }
 
+async function getDirectLineTokenFromSecret(channelSecret) {
+  const bearerToken = "Bearer " + channelSecret
+  const res = await fetch("https://directline.botframework.com/v3/directline/tokens/generate", {
+    headers: {
+      'Authorization': bearerToken
+    },
+    method: 'POST'
+  });
+
+  if (!res.ok) {
+    throw new Error(`Failed to fetch authorization token for Direct Line, server returned ${res.status}`);
+  }
+
+  const { token: directLineToken } = await res.json();
+  return directLineToken;
+}
+
 let credentialsPromise;
 
-export default async function fetchSpeechCredentialsWithCache() {
+export default async function fetchSpeechCredentialsWithCache(enableInternalHttpSupport = false) {
```

Comment:
Just like `createTestHarness` lets put `enableInternalHttpSupport` in an options object.

```js
fetchSpeechCredentialsWithCache(options: { enableInternalHttpSupport = false }): Promise<>;
```

```suggestion
export default async function fetchSpeechCredentialsWithCache({ enableInternalHttpSupport = false }) {
```